docs/machine-learning/tutorials/image-classification.md

```diff
@@ -111,336 +111,249 @@ toaster2.png	appliance
 
-To add the training algorithm, call the `mlContext.MulticlassClassification.Trainers.LbfgsMaximumEntropy()` wrapper method.  The [LbfgsMaximumEntropy](xref:Microsoft.ML.Trainers.LbfgsMaximumEntropyMulticlassTrainer) is appended to the `estimator` and accepts the Inception image features (`softmax2_pre_activation`) and the `Label` input parameters to learn from the historic data.  Add the trainer with the following code:
+1. Add the estimator to load the TensorFlow model, and score it:
 
-[!code-csharp[AddTrainer](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#AddTrainer)]
+    [!code-csharp[ScoreTensorFlowModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ScoreTensorFlowModel)]
 
-You also need to map the `predictedlabel` to the `predictedlabelvalue`:
+    The next stage in the pipeline is to load the TensorFlow model into memory, then process the vector of pixel values through the TensorFlow model network. Applying inputs to a deep learning model is referred to as **Scoring**. When using the model in its entirety, scoring makes an inference, or prediction. 
 
-[!code-csharp[MapValueToKey2](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapValueToKey2)]
+    In this case, you use all of the TensorFlow model except the last layer, which is the layer that makes the inference. The output of the penultimate layer is labeled `softmax_2_preactivation`. This output of this layer is effectively a vector of features that characterize the original input images.
 
-The `Fit()` method trains your model by transforming the dataset and applying the training. Fit the model to the training dataset and return the trained model by adding the following as the next line of code in the `ReuseAndTuneInceptionModel()` method:
+    This feature vector generated by the TensorFlow model will be used as input to the ML.NET trainer below.
 
-[!code-csharp[TrainModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#TrainModel)]
+1. Add the estimator to map the string labels in the training data to integer key values:
 
-The [Transform()](xref:Microsoft.ML.ITransformer.Transform%2A) method makes predictions for multiple provided input rows of a test dataset.  Transform the `Training` data by adding the following code to `ReuseAndTuneInceptionModel()`:
+    [!code-csharp[MapValueToKey](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapValueToKey)]
 
-[!code-csharp[TransformData](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#TransformData)]
+    The ML.NET trainer that is appended next requires its labels to be in `key` format rather than arbitrary strings. A key is a number that has a one to one mapping to a string value.
```

Comment:
Suggestion: Add xref https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.data.keydataviewtype?view=ml-dotnet for `KeyDataViewType`

Reply:
Yeah, I thought about that. But that API documentation is not great atm.